examples/set-annotation/example-config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
I suggest better file names than `example-config.yaml`, otherwise everything is an example config.

Reply:
renamed files